.travis.yml

```diff
@@ -2,12 +2,15 @@ os:
   - linux
   - osx
 language: rust
+before_script:
+  - rustup component add rustfmt-preview
 rust:
   - stable
   - beta
   - nightly
 sudo: false
 script:
+  - cargo fmt --all -- --check
```

Comment:
this is wrong since it runs fmt on both stable & nightly. Fmt should be run only on nightly

Reply:
How do you do that in travis?

Reply:
`if [ "$TRAVIS_RUST_VERSION" = "nightly" ]; then BLAHBLAH; fi`

Reply:
no you don't need to do that. You have to create separate matrices. Look at [this](https://github.com/uuid-rs/uuid/blob/master/.travis.yml) for an example.

Reply:
@Dylan-DPC do you have the time to do these changes?, I won't have time do do anything now for a while.